test/local_integration_test.py

```diff
@@ -59,8 +63,14 @@ def test_webservice_and_indexer(self):
         self.check_endpoint_is_working(config.service_endpoint(), '/version')
         self.check_endpoint_is_working(config.service_endpoint(), '/repository/summary')
         self.check_endpoint_is_working(config.service_endpoint(), '/repository/files/order')
-        manifest_filter = {"file": {"organPart": {"is": ["temporal lobe"]}, "fileFormat": {"is": ["bai"]}}}
+        if config.dss_query_prefix is not '':  # only subset of indexed metadata, use less stringent filter
```

Comment:
```suggestion
        if config.dss_query_prefix:  # only subset of indexed metadata, use less restrictive filter
```

You shouldn't compare strings with `is`. It works for empty and single-character strings but that isn't documented.

http://guilload.com/python-string-interning/

Reply:
Fixed, not sure what I was thinking...